<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transaction</class>
 <widget class="QDialog" name="Transaction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelSellingPrice">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTransaction">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Transaction</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPurchasePrice">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="displayPurchase">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Purchase Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="displaySelling">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Selling Price:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditMonth">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>MMMM yyyy </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
